libraries/botframework-schema/src/index.ts

```diff
@@ -1947,6 +1947,7 @@ export enum ActivityTypes {
     Suggestion = 'suggestion',
     Trace = 'trace',
     Handoff = 'handoff',
+    Delay = 'delay',
```

Comment:
`'delay'` is a "fake" activity type and is only supported by the BotFrameworkAdapter. Can you roll this back?

For reference, the official Activity types are defined in the [Bot Framework Activity specification](https://github.com/microsoft/botframework-obi/blob/main/protocols/botframework-activity/botframework-activity.md).